test/cmd/templates.sh

```diff
@@ -46,6 +46,13 @@ guestbook_template="${OS_ROOT}/test/templates/testdata/guestbook.json"
 os::cmd::expect_success "oc process -f '${guestbook_template}' -l app=guestbook | oc create -f -"
 os::cmd::expect_success_and_text 'oc status' 'frontend-service'
 echo "template+config: ok"
+
+os::test::junit::declare_suite_start "cmd/templates/local-config"
+# Processes the template locally
+os::cmd::expect_success_and_text "oc process -f '${guestbook_template}' --local -l app=guestbook -o yaml" "app: guestbook"
+# Does not even try to hit the server
+os::cmd::expect_success_and_text "oc process -f '${guestbook_template}' --local -l app=guestbook -o yaml --server 0.0.0.0:1" "app: guestbook"
```

Comment:
how about processing the same template twice, once locally and once remotely, and verifying the result is identical?  (can't have generated parameters, of course)